index.html

```diff
@@ -4221,11 +4221,18 @@ <h3>HTML Attribute State and Property Mappings</h3>
               <td class="elements">
                 <a data-cite="html/interaction.html#attr-contenteditable">HTML elements</a>
               </td>
-              <td class="aria">?</td>
+              <td class="aria">
+                <p class="general">
+                  If set to an element which already exposes a corresponding role, then no specific ARIA mapping.
+                </p>
+                <p>
+                  If set to an element with no corresponding role, then map to the <a class="core-mapping" href="#role-map-group">`group`</a> role.
```

Comment:
```suggestion
                  If set on an element with no role, then map to the <a class="core-mapping" href="#role-map-group">`group`</a> role.
```

Same Notes as above.